### PR TITLE
Filter count feature

### PR DESCRIPTION
**Pull request by kayceeDev**
**Description**

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Gatsby/src/pages/catalog.js [hunk 1]
```diff
@@ -8,6 +9,7 @@ import Features from "../components/Catalog/Features/features"
 
 const MembersPage = () => {
   const [filter, setFilter] = useState([]);
+  const [categories] = useState(DataFile)
```
Comment (YashKamboj) [inline Gatsby/src/pages/catalog.js:12]: @kayceeDev It would be better to use another naming here instead of categories

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: Gatsby/src/pages/catalog.js (hunk 1 above)
Comment (kayceeDev) [reply Gatsby/src/pages/catalog.js:12]: What do you suggest?

Comment (YashKamboj) [reply Gatsby/src/pages/catalog.js:12]: "dataFile" can be a good option

Comment (YashKamboj) [reply Gatsby/src/pages/catalog.js:12]: @kayceeDev

Comment (kayceeDev) [reply Gatsby/src/pages/catalog.js:12]: ok. I will send a PR now

**Review by YashKamboj — APPROVED** [review]
LGTM!!!

**Pull request merged**